tests/test_vagrant.py

```diff
@@ -603,6 +608,19 @@ def test_provisioning(vm_dir):
     assert test_file_contents == "foo", "The test file should contain 'foo'"
 
 
+@pytest.mark.parametrize("vm_dir", (SHELL_PROVISION_FAIL_VAGRANTFILE,), indirect=True)
+def test_provisioning_no_destory_on_error(vm_dir):
```

Comment:
You're adding a new test so increase ``PYTEST_REQPASS`` in ``.github/workflows/tox.yml``